src/coreclr/inc/check.inl

```diff
@@ -220,6 +220,14 @@ inline CHECK CheckOverflow(const void *address, UINT64 offset)
     CHECK_OK;
 }
 
+#if defined(HOST_UNIX) && defined(HOST_64BIT)
```

Comment:
Is the `HOST_UNIX` important here?

Reply:
Same question for many of these.

Reply:
Yes, on Windows 64 bit, the SIZE_T is equivalent to INT64, so the compiler complains about multiply defined functions, since there already is a version for INT64.